README.md

```diff
@@ -36,6 +36,20 @@ docker-compose build
 docker-compose up
 ```
 
+### Dashboard
+Plotly dashboard is integrated as a playground for the audio synthesis. In the dashboard, the following options can be configured and new audio synthesized:
+- Number of people in the crowd
+- Female vs. male ratio of the crowd
+- Individual intensity level of clapping, laughing, whistling and booing
+
+```shell
+# Change directory to the project root
+cd audience-synthesis-playground
+
```

Comment:
You should mention to install the requirements

Reply:
done, just added the package installment step to the README